src/test/java/org/prebid/server/bidder/smartadserver/SmartadserverBidderTest.java

```diff
@@ -81,6 +83,30 @@ public void makeHttpRequestsShouldCreateCorrectURL() {
                 .isEqualTo("https://test.endpoint.com/path/api/bid?testParam=testVal&callerId=5");
     }
 
+    @Test
+    public void makeHttpRequestsShouldMergeExistingPublisherData() {
```

Comment:
Actually this test also checks that we update Site object, so it will be great to rename it like
`makeHttpRequestsShouldUpdateSiteObjectIfPresent` or another one